themes/bootstrap3/templates/record/addrelais.phtml

```diff
@@ -0,0 +1,163 @@
+<style>
+
+.glyphicon.spinning {
+    animation: spin 1s infinite linear;
+    -webkit-animation: spin2 1s infinite linear;
+}
+
+@keyframes spin {
+    from { transform: scale(1) rotate(0deg); }
+    to { transform: scale(1) rotate(360deg); }
+}
+
+@-webkit-keyframes spin2 {
+    from { -webkit-transform: rotate(0deg); }
+    to { -webkit-transform: rotate(360deg); }
+}
+
+</style>
+
+
+
+ <div class="row">
+   <div class="col-md-5">
+     <img alt="PALCI EZBorrow" src="<?=$this->imageLink('ezborrowlogo.gif')?>">
+   </div>
+   <div class="col-md-7">
+     <div class="form-group" id="relaisResults">
+        <button class="btn btn-lg btn-info">
+          <span class="glyphicon glyphicon-refresh spinning"></span> Checking availability...    
+        </button>
+      </div>
+    </div>
+  </div>
+
+
+    <div class="row">
+     <div class="col-md-12" id="requestMessage">
+
+
+     </div>
+  </div>
+
+  <div class="row">
+     <div class="col-md-6">
+
+
+
+     </div>
+     <div class="col-md-6" id="requestButton">
+       
+     </div>
+  </div>
+
+
+  <script>
+  $(document).ready(function(){
```

Comment:
It would probably make sense to move this logic to a relais.js file, and include it with the headScript helper at the top of this template.

Reply:
@demiankatz thank you!